bigtable-client-core-parent/bigtable-hbase-integration-tests-common/src/test/java/com/google/cloud/bigtable/hbase/TestScan.java

```diff
@@ -16,13 +16,17 @@
 package com.google.cloud.bigtable.hbase;
 
 import static com.google.cloud.bigtable.hbase.test_env.SharedTestEnvRule.COLUMN_FAMILY;
+import static com.google.cloud.bigtable.hbase.test_env.SharedTestEnvRule.COLUMN_FAMILY2;
+import static org.junit.Assert.assertTrue;
 
 import com.google.protobuf.ByteString;
 import java.io.IOException;
+import java.sql.Date;
```

Comment:
sql date?

Reply:
I am sorry, I may have used `sql.Date` for `Date#valueOf`, I have now removed this import.